AK Apolo/app/src/main/java/ayds/apolo/songinfo/home/view/SongDescriptionHelperImpl.kt

```diff
@@ -3,12 +3,13 @@ package ayds.apolo.songinfo.home.view
 import ayds.apolo.songinfo.home.model.entities.EmptySong
 import ayds.apolo.songinfo.home.model.entities.Song
 import ayds.apolo.songinfo.home.model.entities.SpotifySong
+import ayds.apolo.songinfo.home.utils.FormatterFactory
 
 interface SongDescriptionHelper {
     fun getSongDescriptionText(song: Song = EmptySong): String
 }
 
-internal class SongDescriptionHelperImpl : SongDescriptionHelper {
+internal class SongDescriptionHelperImpl(private val formatterFactory: FormatterFactory) : SongDescriptionHelper {
```

Comment:
Esta linea les queda muy larga. Usen este formato

```suggestion
internal class SongDescriptionHelperImpl(
    private val formatterFactory: FormatterFactory
): SongDescriptionHelper {
```